resources/views/app.blade.php

```diff
@@ -5,6 +5,29 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
     <link href="{{ mix('/css/app.css') }}" rel="stylesheet" />
     <script src="{{ mix('/js/app.js') }}" defer></script>
+
+    <noscript>
```

Comment:
My personal preference would be to include this in a separate template that gets included here. less clutter

Reply:
You mean having a separate `noscript.blade.php` file right? We already have a separate `scss` file for that, so it sounds better for me as well. If that's the case, just let me know so I can update the code.

Reply:
yes that's exactly what i mean!